packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -165,7 +169,17 @@ const deployCoreContract = async (
     )
     if (initializeAbi) {
       const args = initializationData[contractName]
-      const callData = web3.eth.abi.encodeFunctionCall(initializeAbi, args)
+      let callData
+      try {
+        callData = web3.eth.abi.encodeFunctionCall(initializeAbi, args)
+      } catch (error) {
+        console.error(
+          `Tried to initialize new implementation of ${contractName} with args: ${JSON.stringify(
+            args
+          )}. Initialization ABI spec is: ${JSON.stringify(initializeAbi.inputs)}. \nAborting ...`
+        )
+        throw new Error()
```

Comment:
We log useful info here but then throw an empty error up the callstack here. Is this desired?
Also, the log says "Aborting" whereas this error could be caught further up the callstack and the script wouldn't abort?